### PR TITLE
Copy fix

### PR DESCRIPTION
**Pull request by gustaveen**
## What?

- Update opening hours
- Correct link on DK forever page

### CODE REVIEW

**Review by robinandeer — DISMISSED** [review]

File: src/translations/da_DK.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "ONBOARDING_CONNECT_DD_PAGE_TITLE": "Tilknyt automatisk betaling | Hedvig",
   "COVERAGE_LABEL": "Dækning",
   "COVERAGE_HEADLINE": "Omfattende forsikringsdækning",
-  "COVERAGE_BODY": "Al dækning du mangler lige ved dine fingre. Kundeservice med rigtige mennesker fra 09-22 hver dag. Hurtig forsikringskrav håndteres i appen. Lav selvrisiko på 500 DKK, ingen selvrisiko for store forsikringskrav. Klik på ikonet for mere information.",
+  "COVERAGE_BODY": "Al den dækning du har brug for lige ved hånden. Kundeservice med rigtige mennesker til 22 hver dag. Hurtig håndtering af forsikringskrav i appen. Lav selvrisiko på 500 kr, ingen selvrisiko for store forsikringskrav. Klik på ikonerne nedenfor for mere information.",
```
Comment (styrelius) [inline src/translations/da_DK.json:23]: Shouldn't it still say that it's from 9 AM?

Comment (gustaveen) [reply src/translations/da_DK.json:23]: They changed this back and forth since it's 9 on weekdays and from 10 on weekends :)

Comment (gustaveen) [reply src/translations/da_DK.json:23]: But it might be that they have changed it again 😁

Comment (styrelius) [reply src/translations/da_DK.json:23]: Haha, OK, let's leave it like this then 😄

**Review by styrelius — DISMISSED** [review]
🦚

**Review by robinandeer — APPROVED** [review]

**Pull request merged**